src/AspNetWebApi/WebApi/Controllers/GraphQLController.cs

```diff
@@ -54,10 +48,10 @@ public async Task<HttpResponseMessage> PostAsync(HttpRequestMessage request, Gra
                 ? HttpStatusCode.BadRequest
                 : HttpStatusCode.OK;
 
-            var json = await _writer.WriteToStringAsync(result);
```

Comment:
`var httpResult = result.Errors?.Count > 0` condition above should be changed like in other projects to use `result.Executed` property.

Reply:
Done